internal/server/api.go

```diff
@@ -56,7 +56,7 @@ type HTTPServerError struct {
 }
 
 // NewService creates a service struct, containing all that is needed for a mmsd server to run.
-func NewService(templates *template.Template, eventsDB *sql.DB, stateDB *sql.DB, natsURL string) *Service {
+func NewService(templates *template.Template, eventsDB *sql.DB, stateDB *sql.DB, natsURL string, productDropTimeout int) *Service {
```

Comment:
I think its time to consider a config struct or something for `NewService`, as the parameter list to the function is getting pretty long.

Reply:
Agree, I felt I was trying to push it a bit too far yesterday, now I have it confirmed :)